sdk/src/main/java/io/opentelemetry/sdk/resources/EnvVarResource.java

```diff
@@ -18,6 +18,8 @@
 
 import com.google.common.annotations.VisibleForTesting;
 import io.opentelemetry.common.Attributes;
+import io.opentelemetry.sdk.common.export.ConfigBuilder;
```

Comment:
We most likely need to later update the code for `ConfigBuilder` to live right under `common`, instead of `common.export`, as this configuration pattern is useful in other scenarios (cc @thisthat )